src/libraries/System.Net.Security/src/System/Net/Security/Pal.Android/SafeFreeSslCredentials.cs

```diff
@@ -0,0 +1,47 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net
+{
+    internal sealed class SafeFreeSslCredentials : SafeFreeCredentials
```

Comment:
Looks like exact copy of macOS. Perhaps we can share it...?

Reply:
Thoughts on location/naming for sharing something that is 'Android and OSX, but not other Unix'?

Reply:
I put it in the `Pal.Managed` directory, since the other `SafeFreeSslCredentials` implementation relies on P/Invokes and this one does not.